src/odemis/dataio/tiff.py

```diff
@@ -1727,6 +1733,23 @@ def _reconstructFromOMETIFF(xml, data, basename):
 
     return omedata
 
+def _gen_resized_shapes(original_shape):
+    """
+    Generates a list of tuples with the size of the resized images
+    original_shape (tuple of shape YX or YXC): shape of the original image
+    return (list of tuples): List of the tuples with the size of the resized images
+    """
+    shape = original_shape
+    resized_shapes = []
+    z = 0
+    while shape[0] >= TILE_SIZE and shape[1] >= TILE_SIZE:
```

Comment:
I also changed the name of the _gen_resized_shapes to _genResizedShapes to follow the file's convention